lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -249,7 +249,7 @@ def test_pcolormesh_global_with_wrap1():
 @ImageTesting(['pcolormesh_global_wrap2'])
 def test_pcolormesh_global_with_wrap2():
     # make up some realistic data with bounds (such as data from the UM)
-    nx, ny = 36, 18
+    nx = 36
     xbnds, xstep = np.linspace(0, 360, nx - 1, retstep=True, endpoint=True)
     ybnds, ystep = np.linspace(-90, 90, nx - 1, retstep=True, endpoint=True)
```

Comment:
Seems better to make this `ny` (but that would require changing `ny` since it was not used before.)

Reply:
Really probably just `n`, but I don’t think it’s really worth worrying about.

Reply:
Ok, yes I hadn't spotted it -- it looks like that was the error.
Fixed..